### PR TITLE
Update workspace tooltip

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes 

https://issues.redhat.com/browse/KFLUXBUGS-1492

## Description

Updates User access tooltip in GrantAccess Page

## Type of change

- [ ] Feature
- [x] Bugfix
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Screen shots / Gifs for design review 

Screenshot 2024-08-05 at 5 43 58 PM

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/UserAccess/UserAccessForm/UsernameSection.tsx [hunk 1]
```diff
@@ -68,9 +68,9 @@ export const UsernameSection: React.FC<React.PropsWithChildren<Props>> = ({ disa
         label="Enter usernames"
         labelIcon={
           <HelpPopover
-            aria-label="Usernames in RHTAP"
-            headerContent="Usernames in RHTAP"
-            bodyContent="To find their username, the user you want to invite should log into RHTAP and click Workspaces. Their username is the first item in the breadcrumb navigation."
+            aria-label="Usernames in KONFLUX"
```
Comment (sahil143) [inline src/components/UserAccess/UserAccessForm/UsernameSection.tsx:71]: change at other places also
```suggestion
            aria-label="Usernames in Konflux"
```

**Review by sahil143 — CHANGES_REQUESTED** [review]

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Pull request merged**